packages/wonder-blocks-button/components/button-core.js

```diff
@@ -60,11 +67,15 @@ export default class ButtonCore extends React.Component<Props> {
     }
 }
 
+const overridableStyles = {
```

Comment:
Why pull these out of `sharedStyles.shared`?

Reply:
If we leave them in, Aphrodite adds "!important" to them, which you can't override with passed in styles (I think). I could also leave them in the same object and pull them all out of Aphrodite styling, but then the button could accept any styling (until ADR 47?)

Reply:
https://github.com/Khan/wonder-blocks/pull/79 addresses the issue.

Reply:
Question: do we actually want anyone to be able to override these styles? It seems like they would be quite important to leave intact! (Otherwise the layout of the button will get messed up)

Reply:
The plan was to use static type checking to disallow styles that shouldn't be override so that we could override some style props while not others.